### PR TITLE
Admin login Feature added

### PR DESCRIPTION
**Pull request by Christykumar**
- [x] Admin Login feature added
- [x]Validations tested

### CODE REVIEW

File: src/main/java/in/bloomapp/userservice/AdminLogin.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	private AdminLogin() {
+		
+	}
+	/**
+	 * Checks weather the user name and password is valid
+	 * @param username
+	 * @param password
+	 * @return
+	 */
+	public static boolean login(String username, String password) {
```
Comment (selva08sankari) [inline src/main/java/in/bloomapp/userservice/AdminLogin.java]: how admin login comes under userservice? user & admin are same

File: src/test/java/in/bloomapptest/validator/FlowerIsExistTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		assertEquals(null,available);
+	}
+	
+	@Test
+	public void invalidFlowerTypeTest() {
+		Flower available=Validator.flowerIsExist("Natural", "rose");
+		assertEquals(null,available);
+	}
+
+}
```
Comment (selva08sankari) [inline src/test/java/in/bloomapptest/validator/FlowerIsExistTest.java:36]: where is admin login test?

Comment (Christykumar) [reply src/test/java/in/bloomapptest/validator/FlowerIsExistTest.java:36]: will add

File: src/main/java/in/bloomapp/userservice/AdminLogin.java (hunk 1 above)
Comment (Christykumar) [reply src/main/java/in/bloomapp/userservice/AdminLogin.java]: will change

**Pull request merged**